packages/ui/src/styles.css

```diff
@@ -1,4 +1,9 @@
 @import url('./shared-style-variables.css');
+@import url('./remove-old-theme-values.css');
+
+@import url('@harnessio/core-design-system/core-styles');
+@import url('@harnessio/core-design-system/themes');
+@import url('@harnessio/core-design-system/mfe-themes');
```

Comment:
This is there section where we are adding new design themes.

Soon we will move `@harnessio/core-design-system/themes` only to standalone application and `@harnessio/core-design-system/mfe-themes` shared only to MFE.